biztalk/core/office365-mail-adapter.md

```diff
@@ -63,18 +63,22 @@ The Office 365 Outlook Email Adapter allows you to send and receive mails from y
 
 1. The default properties may also be updated using promoted properties:
 
-|Property|Promoted property|
-|---|---|
-| **To** | OfficeMail.To |
-| **CC** | OfficeMail.CC |
-| **Subject** | OfficeMail.Subject |
-| **Importance** | OfficeMail.Importance |
+    Property|Promoted property|
+    |---|---|
+    | **To** | OfficeMail&#46;To |
+    | **CC** | OfficeMail&#46;CC |
+    | **Subject** | OfficeMail.Subject |
+    | **Importance** | OfficeMail.Importance |
+    | **Attached&nbsp;Files** | OfficeMail.AttachedFiles |
+
+
+2. The *OfficeMail.AttachedFiles* property value is list of files with "|" as delimiter in between different files. For instance: "C:\\Attachments\File1.pdf|C:\\Attachments\File2.xml".

```

Comment:
when you're quoting property names, either use bold (**OfficeMail.AttachedFiles**) or back ticks (`OfficeMail.AttachedFiles`)

Reply:
done